python-api/app/integrations/openrouter/client.py

```
@@ -1,180 +1,440 @@
+            response_data = asyncio.run(self.http_client.get(
+                f"/models/{model_id}/endpoints"
+            ))
+            
+            # Handle both array and object responses
+            if isinstance(response_data, list):
+                endpoints_data = response_data
+            else:
+                endpoints_data = response_data.get("data", [])
+                
+            # endpoints = [ModelEndpoint(**endpoint) for endpoint in endpoints_data]
+            endpoints = [ModelEndpoint(**endpoints_data)]
```

Ruff (0.12.2)

Local variable `endpoints` is assigned to but never used

Remove assignment to unused variable `endpoints`

(F841)